src/main/kotlin/no/nav/syfo/application/ApplicationEnvironment.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.syfo.application
+
+data class Environment(
+    val namespace: String = "teamsykefravr",
+    val appname: String = "fastlegerest",
+    val aadAppClient: String = getEnvVar("AZURE_APP_CLIENT_ID"),
```

Comment:
En mulig forbedring kan være samle disse i en AzureEnvironment. Og tilsvarende ClientsEnvironment for syfotilgingskontroll osv under, som vi har gjort i noen andre apper.